modules/registrars/openprovider/constants/Constants.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace OpenProvider\WhmcsRegistrar\Constants;
+
+use Carbon\Carbon;
+
+class Constants
+{
+    public static function getAuthTokenExpirationTimeFromNow()
```

Comment:
No, I mean const like 
`const SESSION_EXPIRATION_LIFE_TIME = 300;`
In openprovider/init.php